docs/REST_API.md

```diff
@@ -331,7 +331,7 @@ _**Example Response**_
 }
 ```
 ## Task API
-snap task APIs provide the functionality to create, start, stop, remove, enable, retrieve and watch scheduled tasks. 
+Snap task APIs provide the functionality to create, start, stop, remove, enable, retrieve and watch scheduled tasks. 
```

Comment:
No sign of export in this doc at all. Would you take it as a separate task to review the accuracy of the API doc @tiffanyfj ?